tests/epyccel/test_epyccel_python_annotations.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+from pyccel.epyccel import epyccel
+
+def test_module_1():
+    import modules.python_annotations as mod
+
+    modnew = epyccel(mod)
+    assert modnew.fib(10) == mod.fib(10)
```

Comment:
Why are you only using one of the test functions you wrote?

Reply:
I only test the compilation, there is no need to test the results

Reply:
It doesn't do any harm though, just in case the types are misinterpreted. Even if I don't see how they could be

Reply:
okay

Reply:
done